homework/02LessonTwo.js

```diff
@@ -53,23 +63,27 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 //
 // What is a function? How do you define a function in Javascript?
 //
-// Your Answer Goes Here:
-//
+// Your Answer Goes Here: A function is a collection of commands that you can
+// use to keep from typing the same block(s) of code over and over.
+// A function is defined like so: function() {}
 //
 // What is a return value?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: A return value is a parameter returned by a function.
```

Comment:
I wouldn't say that the return value is a parameter. I would probably say, that the return value is what a function call evaluates to.